Tella/Data/Database/TellaDataBase.swift

```diff
@@ -26,14 +26,18 @@ class TellaDataBase : DataBase {
             case 0:
                 createTables()
             case 1:
+                alterTable()
```

Comment:
we don't need to update the version 1 we need to keep it because this is already uploaded to the appstore